styles/gemini/catppuccin.user.css

```diff
@@ -0,0 +1,160 @@
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('gemini.google.com') {
+  :root body.dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  :root body.light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @subtext0-rgb: red(@subtext0), green(@subtext0), blue(@subtext0);
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @mantle-rgb: red(@mantle), green(@mantle), blue(@mantle);
+    @crust: @catppuccin[@@lookup][@crust];
+    @crust-rgb: red(@crust), green(@crust), blue(@crust);
+    @accent-color: @catppuccin[@@lookup][@@accent];
+    @accent-color-rgb: red(@accent-color), green(@accent-color),
+      blue(@accent-color);
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --bard-color-primary: @accent-color;
+    --bard-color-on-primary-container: @text;
+    --bard-color-surface-container: @base;
+    --bard-color-surface-container-low: lighten(@mantle, 2%);
+    --bard-color-surface-container-highest: lighten(@base, 5%);
+    --bard-color-surface-container-high: @surface0;
+    --bard-color-inverse-on-surface: darken(@mantle, 10%);
+    --bard-color-outline-variant: @surface1;
+    --bard-color-main-container-background: @mantle;
+    --bard-color-main-container-background-rgb: @mantle-rgb;
```

Comment:
These should be `base`? The main background is always base. I would write a suggestion for you but since you aren't using the rgbify utility... 🫠